oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/StorageModuleElasticsearchConfig.java

```diff
@@ -20,6 +20,7 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import org.apache.skywalking.oap.server.core.storage.annotation.Column;
```

Comment:
This is illegal import.

Reply:
it's used in link, may not understand your suggestion to modify this.
![image](https://user-images.githubusercontent.com/31562192/105635106-3f9b8d00-5e9c-11eb-90cb-a4a95787b0f1.png)

Reply:
I think you will see `maven` failure. You need to use the full name in the `@link`.